docs/template_plugin/tests/functional/shared_tests_instances/single_layer_tests/softmax.cpp

```diff
@@ -11,34 +12,63 @@ using namespace LayerTestsDefinitions;
 
 namespace {
 
-const std::vector<InferenceEngine::Precision> netPrecisions = {
+const std::vector<InferenceEngine::Precision> netPrecision = {
     InferenceEngine::Precision::FP32,
 };
 
 const std::vector<InferenceEngine::Layout> inputLayouts2D = {
     InferenceEngine::Layout::NC,
 };
 
-const std::vector<InferenceEngine::SizeVector> inputShapes2D = {
-    InferenceEngine::SizeVector {1, 100},
-    InferenceEngine::SizeVector {100, 1},
-    InferenceEngine::SizeVector {10, 10},
+const std::vector<std::vector<std::pair<size_t, size_t>>> inputStaticShape2D = {
+    {NULL_RANGE}
+};
+
+const std::vector<std::vector<std::pair<size_t, size_t>>> inputShape2D = {
```

Comment:
inputDynamic

Reply:
Added dynamic case, but I'm not sure if it meets your expectation. Could you check it please?